libs/librepcbcommon/systeminfo.cpp

```diff
@@ -24,6 +24,12 @@
 #include <QHostInfo>
 #include "systeminfo.h"
 
+#if (defined(Q_OS_UNIX) || defined(Q_OS_LINUX)) && (!defined(Q_OS_MACX)) // For UNIX and Linux
+#include <sys/types.h>
```

Comment:
I copied it from the manpage. But you're right, it doesn't seem to be necessary. I'll remove it.